<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dashBoard</class>
 <widget class="QMainWindow" name="dashBoard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="email">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>10</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>abc@gmail.com</string>
    </property>
   </widget>
   <widget class="QLabel" name="email_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>10</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>Welcome,</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Serif</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Library Dashboard</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_Items">
    <property name="title">
     <string>Add Items</string>
    </property>
    <addaction name="addItems"/>
    <addaction name="actionCourse"/>
    <addaction name="actionDepartment"/>
   </widget>
   <widget class="QMenu" name="menuBook">
    <property name="title">
     <string>Book</string>
    </property>
    <addaction name="book"/>
    <addaction name="actionPurchase"/>
   </widget>
   <widget class="QMenu" name="menuActivities">
    <property name="title">
     <string>Activities</string>
    </property>
    <addaction name="activities"/>
    <addaction name="actionIssue_aa_book"/>
    <addaction name="actionReturn_book"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
    <widget class="QMenu" name="report">
     <property name="title">
      <string>Purchase</string>
     </property>
     <addaction name="actionAll"/>
     <addaction name="actionMonthly"/>
    </widget>
    <widget class="QMenu" name="menuIssue">
     <property name="title">
      <string>Issue</string>
     </property>
     <addaction name="actionAll_2"/>
     <addaction name="actionMonthly_2"/>
    </widget>
    <widget class="QMenu" name="menuReturn">
     <property name="title">
      <string>Return</string>
     </property>
     <addaction name="actionAll_3"/>
     <addaction name="actionMonthly_3"/>
    </widget>
    <widget class="QMenu" name="menuFine">
     <property name="title">
      <string>Fine</string>
     </property>
     <addaction name="actionAll_4"/>
     <addaction name="actionMonthly_4"/>
    </widget>
    <addaction name="report"/>
    <addaction name="menuIssue"/>
    <addaction name="menuReturn"/>
    <addaction name="menuFine"/>
    <addaction name="actionStudent"/>
    <addaction name="actionBook_Stock"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="settings"/>
    <addaction name="actionProfile"/>
   </widget>
   <widget class="QMenu" name="menuAdd_user">
    <property name="title">
     <string>Add user</string>
    </property>
    <addaction name="addUser"/>
    <addaction name="actionStudent_2"/>
   </widget>
   <addaction name="menuAdd_Items"/>
   <addaction name="menuAdd_user"/>
   <addaction name="menuBook"/>
   <addaction name="menuActivities"/>
   <addaction name="menuReport"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="addItems">
   <property name="text">
    <string>Subject</string>
   </property>
  </action>
  <action name="actionCourse">
   <property name="text">
    <string>Course</string>
   </property>
  </action>
  <action name="actionDepartment">
   <property name="text">
    <string>Department</string>
   </property>
  </action>
  <action name="book">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionPurchase">
   <property name="text">
    <string>Purchase</string>
   </property>
  </action>
  <action name="activities">
   <property name="text">
    <string>Book request</string>
   </property>
  </action>
  <action name="actionIssue_aa_book">
   <property name="text">
    <string>Issue a book</string>
   </property>
  </action>
  <action name="actionReturn_book">
   <property name="text">
    <string>Return book</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="text">
    <string>All purchase</string>
   </property>
  </action>
  <action name="actionMonthly">
   <property name="text">
    <string>Monthly purchase</string>
   </property>
  </action>
  <action name="actionAll_2">
   <property name="text">
    <string>All issue</string>
   </property>
  </action>
  <action name="actionMonthly_2">
   <property name="text">
    <string>Monthly issue</string>
   </property>
  </action>
  <action name="actionStudent_wise">
   <property name="text">
    <string>Student wise</string>
   </property>
  </action>
  <action name="actionAll_3">
   <property name="text">
    <string>All return</string>
   </property>
  </action>
  <action name="actionMonthly_3">
   <property name="text">
    <string>Monthly return</string>
   </property>
  </action>
  <action name="actionAll_4">
   <property name="text">
    <string>All fine</string>
   </property>
  </action>
  <action name="actionMonthly_4">
   <property name="text">
    <string>Monthly fine</string>
   </property>
  </action>
  <action name="actionStudent">
   <property name="text">
    <string>Student Report</string>
   </property>
  </action>
  <action name="settings">
   <property name="text">
    <string>Change Password</string>
   </property>
  </action>
  <action name="actionProfile">
   <property name="text">
    <string>Profile</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="addUser">
   <property name="text">
    <string>Librarian</string>
   </property>
  </action>
  <action name="actionStudent_2">
   <property name="text">
    <string>Student</string>
   </property>
  </action>
  <action name="actionBook_Stock">
   <property name="text">
    <string>Book Stock</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
